packages/cli/src/templates/plugins/SwPluginEmptySlot.vue

```diff
@@ -0,0 +1,3 @@
+<template>
+  <div></div>
```

Comment:
Would really appreciate empty slots to resolve to empty markup, is that an issue? Seems like unnecessary overhead.

Reply:
removed this component :)